konan_cli/main.py

```diff
@@ -29,21 +28,27 @@ def konan(ctx, version):
 
 
 @konan.command()
-@click.option('--email', prompt="Email", help="The email you registered with on Konan", required=True, type=click.STRING)
-@click.option('--password', prompt="Password", help="The password of your registered user on Konan", required=True, hide_input=True, type=click.STRING)
-def login(email, password):
+@click.option('--email', prompt="Email", help="The email you registered with on Konan", required=True,
+              type=click.STRING)
+@click.option('--password', prompt="Password", help="The password of your registered user on Konan", required=True,
```

Comment:
I don't think password should be required here once we enter the api key. The behaviour needs to ensure one of them is set and prompt it if left empty.